modulus/models/layers/fused_silu.py

```diff
@@ -13,16 +13,27 @@
 # limitations under the License.
 
 import functools
+import logging
+from typing import Tuple
 
 import torch
 from torch.autograd import Function
 
+logger = logging.getLogger(__name__)
+
 try:
-    from nvfuser._C import DataType, Fusion, FusionDefinition
-except ImportError:
-    # accomodating for earlier versions of PyTorch (< 2.0)
-    # which don't need nvfuser as explicit dependency
-    from torch._C._nvfuser import DataType, Fusion, FusionDefinition
+    import nvfuser
```

Comment:
Why no change to the `pyproject.toml` with nvfuser in an optional dependency?

https://github.com/NVIDIA/Fuser

Reply:
Similar to DGL, this cannot be added to the pyproject.toml because of PyPi requirements. I have added an instruction to the README as well as our Getting Started guide. https://docs.nvidia.com/deeplearning/modulus/getting-started/index.html